src/api.js

```diff
@@ -39,3 +39,11 @@ export const getCommentByArticleID = (id) => {
 export const postCommentByArticleID = (id, comment) => {
   return api.post(`/articles/${id}/comments`, comment);
 };
+
+export const deleteCommentByID = (comment_id) => {
+  return api
+    .delete(`/comments/${comment_id}`, { params: { comment_id } })
```

Comment:
I don't think you need the {params: {comment_id}} part of this - it should just work being passed the comment id at the end of the endpoint and (confusingly) params is used in axios for adding queries to endpoints.